src/cbmc/bmc.cpp

```diff
@@ -448,6 +388,107 @@ safety_checkert::resultt bmct::execute(const goto_functionst &goto_functions)
   }
 }
 
+safety_checkert::resultt bmct::execute(lazy_goto_modelt &lazy_model)
```

Comment:
This parameter needs to be passed by r-value if you're going to `std::move` it below. i.e. the model needs to be moved into this function, indicating to the caller that it is no longer valid after this function has been called.